sympy/physics/vector/vector.py

```diff
@@ -695,6 +695,62 @@ def applyfunc(self, f):
             d[v[1]] = v[0].applyfunc(f)
         return Vector(d)
 
+    def angle_between(self, vec):
+        """
+        Returns the angle between Vector 'vec' and self.
+
+        Parameter
+        =========
+
+        vec : Vector
+            The Vector between which angle is needed.
+
+        Examples
+        ========
+
+        >>> from sympy.physics.vector import ReferenceFrame
+        >>> A = ReferenceFrame("A")
+        >>> v1 = A.x
+        >>> v2 = A.y
+        >>> v1.angle_between(v2)
+        pi/2
+
+        """
+
+        from sympy.physics.vector.functions import dot
+        vec1 = self.normalize()
+        vec2 = vec.normalize()
+        angle = acos(dot(vec1, vec2))
+        return angle
+
+    def unit_normal_from(self, vec):
```

Comment:
A unit vector normal to both

Maybe this is a more informative name (i.e. just a modified cross product).

vector_a.unit_cross(vector_b)

but this is so trivial. Current code is:

vector_a.cross(vector_b).normalize()

I don't think we really need a method for this, since the existing method is so simple. The angle_between is nice because you don't have to remember the formula, but this one seems unnecessary.